bitbucket-slack-server-integration-plugin/pom.xml

```diff
@@ -407,7 +407,7 @@
                     <skipAllPrompts>true</skipAllPrompts>
 
                     <!-- It allows to connect to Bitbucket embedded database while it is running -->
-                    <jvmArgs>-Djdbc.url=jdbc:h2:${project.build.directory}/bitbucket/home/shared/data/db;MVCC=TRUE;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;TRACE_LEVEL_FILE=4;AUTO_SERVER=TRUE</jvmArgs>
+                    <jvmArgs>-Djdbc.url=jdbc:h2:${project.build.directory}/bitbucket/home/shared/data/db;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE</jvmArgs>
```

Comment:
This is required as Bitbucket 8.0 uses H2 1.4.200 which doesn't allow MVCC setting